app/script/calling/entities/CallEntity.js

```diff
@@ -257,30 +254,40 @@ z.calling.entities.CallEntity = class CallEntity {
 
   /**
    * Join the call.
-   * @returns {undefined} No return value
+   * @param {z.media.MediaType} [mediaType=z.media.MediaType.AUDIO] - Media type of the call
```

Comment:
What's the reason for `[mediaType=z.media.MediaType.AUDIO]`?

Reply:
Oh yeah, the default value has moved to the method `_joinGroupCall`.
https://github.com/wireapp/wire-webapp/pull/3450/files#diff-4816db425e94bac1930aae4a710ce13bR283

You're right, we should restore the default value for this function also :)